tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsWriter.cs

```diff
@@ -48,6 +55,30 @@ internal class DataStreamsWriter
             period: bucketDurationMs);
     }
 
+    /// <summary>
+    /// Public for testing only
+    /// </summary>
+    public event EventHandler<EventArgs>? FlushComplete;
+
+    private enum SupportState
```

Comment:
(non-blocking/style)

Would it be safer/better to include the values here? Just thinking that it might be good since we covert from the actual `enum` to `int` value

Reply:
Generally we avoid that unless they're publicly available. This is all in memory, so there's no risk of changing values in versions causing problems

Reply:
Good to know and makes sense to me